lib/python/rose/reporter.py

```diff
@@ -252,9 +253,11 @@ def is_closed(self):
     def write(self, message):
         """Write the message to the context's handle."""
         try:
-            return self.handle.write(message.encode("utf-8"))
-        except UnicodeDecodeError:
+            return self.handle.buffer.write(message.encode("utf-8"))
+        except TypeError:
             return self.handle.write(message)
+        except AttributeError:
+            return self.handle.write(message.encode('UTF-8'))
```

Comment:
This could be why the stderr is coming at the end of all stdout in rosa db-create @dpmatthews

Reply:
Jamming in a `self.handle.flush()` might to the trick.